src/cdk/drag-drop/directives/drag.ts

```diff
@@ -356,6 +379,33 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
     return element;
   }
 
+  /**
+   *  Gets the element for the preview clone to be created in,
+   *  based on the `previewContainer` value.
+   */
+  private _getPreviewContainer() {
```

Comment:
You should be able to clean it up like this:

```
private _getPreviewContainer() {
  const cloneTarget = this.previewContainer;

  if (!cloneTarget || cloneTarget === 'global') {
    return null;
  }

  if (cloneTarget === 'parent') {
    return this.element.nativeElement.parentElement;
  }

  return coerceElement(cloneTarget);
}
```

Reply:
Sure thing,
First PR so I'd assumed based on other parts of the code that the type check was important stylistically.

I've done the stylistic part of this, holding off on the `dropContainer` bit mentioned in another comment until i can get you some performance related stats.